tools/check/check-tidy.sh

```diff
@@ -22,4 +22,6 @@ set -euo pipefail
 # go mod tidy do not support symlink
 cd -P .
 
+cp go.sum /tmp/go.sum.before
```

Comment:
It would be better to add a timestamp to the file name

Reply:
IMHO, it's not a good choice, many temporary files would stay in `/tmp`.